back/engines/commercial/insights/app/controllers/insights/web_api/v1/inputs_controller.rb

```diff
@@ -12,7 +12,7 @@ def show
       def index
         # index is not policy scoped, instead the view is authorized.
         inputs = Insights::InputsFinder.new(view, index_params).execute
-        render json: serialize(inputs)
+        render json: linked_json(inputs, InputSerializer, params: fastjson_params, include: [:source])
```

Comment:
Just to be sure, what was wrong exactly? I see the links for the pagination were missing, but now categories are no longer included and it includes full versions of the ideas. Is it on purpose?

Reply:
Ah, no good catch, I think it's better now, right ?

Reply:
All good!